docs/usage/upgrade-best-practices.md

```diff
@@ -0,0 +1,274 @@
+
+## General recommendations
+
+In general, you should:
+
+- Run Renovate on _every_ repository
+- Use the `config:best-practices` preset instead of `config:recommended`
+- Use the Dependency Dashboard issue (it's on by default)
+- Update your dependencies often
+- Read the changelogs for the updates
+- Update to new `major` versions in good time
+- Talk with your team about the update stategy
+
+If Renovate is too noisy for you, read the [noise reduction docs](./noise-reduction.md).
+
+## Use the `config:best-practices` preset
+
+The `config:recommended` preset is the recommended configuration for most Renovate users.
+Renovate also has a `config:best-practices` preset that includes our upgrade best practices.
+
+You should extend from the `config:best-practices` preset:
+
+```json
+{
+  "extends": ["config:best-practices"]
+}
+```
+
+If you're using `config:recommended` now, replace it with `config:best-practices`:
+
+```diff
+- "extends": ["config:recommended"]
++ "extends": ["config:best-practices"]
+```
+
+### What's in the `config:best-practices preset?
+
+The [`config:best-practices` preset](https://docs.renovatebot.com/presets-config/#configbest-practices) has this configuration:
+
+```json
+{
+  "configMigration": true,
+  "extends": [
+    "config:recommended",
+    "docker:pinDigests",
+    "helpers:pinGitHubActionDigests",
+    ":pinDevDependencies"
+  ]
+}
+```
+
+The next sections explain each part of the preset.
+
+#### Config migration
+
+Renovate creates a config migration PR to replace old config option names with their new replacements.
+This way your configuration file and the Renovate docs always use the same terms.
+
+#### Extends `config:recommended`
+
+The `config:recommended` preset is a good base to start from.
+That's why we extend from it.
+
+#### Extends `docker:pinDigests`
+
+The [Renovate docs, Docker Digest pinning](./docker.md#digest-pinning) section explains _why_ you should pin your Docker containers to an exact digest.
+
+#### Extends `helpers:pinGitHubActionDigests`
+
+The [GitHub Docs, using third-party actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) recommend that you pin third-party GitHub Actions to a full-length commit SHA.
+
+We recommend pinning _all_ Actions.
+That's why the `helpers:pinGitHubActionDigests` preset pins all GitHub Actions.
+
+#### Extends `:pinDevDependencies`
+
+Pinning your development dependencies means you, and your team, are using the same versions of development tools.
+This makes the developer-tool side of your builds reproducible.
+Debugging faulty versions of your tools is easier, because you can use Git to check out different versions of the tools.
+
+### Why updating often is easier, faster and safer
+
+You may think that updating takes too much time.
+But updating regulary actually _saves_ you time, because:
+
+- Regular updates tend to be small
+- Applying `major` updates is easier
```

Comment:
I realise we've already decided on what `config:best-practices` contains, but is it worth having major and minor split out? Means it's easier to apply `major` when you're ready, while retaining ability to do minor/patch

Reply:
Our [FAQ, default behavior for major/minor releases](https://docs.renovatebot.com/faq/#renovates-default-behavior-for-majorminor-releases) says:

> By default, Renovate separates major and minor releases.

So, just going of that, Renovate should already do what you want? It's not clear to me if we mean "default behavior if no config at all", or "default behavior for `config:recommended`". I'm probably missing something important, because you're asking for a behavior change... 😄